src/server/game/Globals/ObjectMgr.h

```diff
@@ -1403,6 +1415,15 @@ class TC_GAME_API ObjectMgr
             return nullptr;
         }
 
+        SceneTemplate const* GetSceneTemplate(uint32 sceneId) const
+        {
+            auto itr = _sceneTemplateStore.find(sceneId);
+            if (itr != _sceneTemplateStore.end())
```

Comment:
there is a mix of codestyles, sometimes itr is compared with == .end(), sometimes it's compared with != .